src/Components/pages/Form.js

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import Button from './button';
+import { addBook } from '../../redux/books/booksSlice';
 
-const Form = () => (
-  <form className="bookcontainer">
-    <input type="text" placeholder="Book Title" />
-    <input type="text" placeholder="Author" />
-    <Button color="#0290ff" text="Add" />
-  </form>
-);
+const Form = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const min = 1;
+    const max = 100;
+    const randomNum = Math.floor(Math.random() * (max - min + 1)) + min;
+    const formData = { item_id: randomNum + title, title, author };
+    dispatch(addBook(formData));
+    setTitle('');
+    setAuthor('');
```

Comment:
- [ ] [_OPTIONAL_] great job, this is optional but you can make it so that it's impossible to submit an empty form 👍.